full.cljs/src/full/cljs/http.cljs

```diff
@@ -16,13 +16,14 @@
                    :headers headers
                    :format request-format
                    :response-format response-format
-                   :handler (fn [[ok res]]
-                              (put! ch (if ok
-                                         {:status 200
-                                          :body res}
-                                         {:status (:status res)
-                                          :body (:response res)}))
-                              (close! ch))})
+                   :handler (or (response-handler ch)
+                                (fn [[ok res]]
```

Comment:
Why not put this in a named function so it can be tested & reused elsewhere, if needed?